### PR TITLE
Update with Magnes Xcode 15.0 with MinOSVersion 100

### PR DESCRIPTION
**Pull request by KunJeongPark**
### Summary of changes

- Added new version of Magnes to fix Xcode 15.3 SPM bug
  It was built with Xcode 15.0
  MinimumOSVersion in Info.plist is modified to from 12 to 100 manually and framework was re-signed.

Demo app generates privacy items for PPRiskMagnes if you select "Embed and Sign" for the framework
Screenshot 2024-04-09 at 1 45 12 PM

### Checklist

- [x] Added a changelog entry

- @KunJeongPark

### CODE REVIEW

**Comment by KunJeongPark** [comment]
client-hosted-libraries PR: https://github.braintreeps.com/braintree/hosted-client-libraries/pull/164

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * Require Xcode 15.0+ and Swift 5.9+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
 * Updated expiring pinned vendor SSL certificates
+* BraintreeDataCollector
+  * Bump to PPRiskMagnes v5.5.0-static-Xcode15-MinOSVersion100 with fix for Xcode 15.3 SPM validation bug
```
Comment (jaxdesmarais) [inline CHANGELOG.md:7]:
```suggestion
  * Bump to PPRiskMagnes version 5.5.0 with fix for Xcode 15.3 Swift Package Manager validation bug (fixes #1229)
```

**Review by jaxdesmarais — COMMENTED** [review]
Is this version missing the privacy manifest or is that just not showing in the diff. Just want to be sure.

File: CHANGELOG.md (hunk 1 above)
Comment (KunJeongPark) [reply CHANGELOG.md:7]: Yes I think it's just missing in the diff.  I see the Privacy Manifest file.

**Review by jaxdesmarais — APPROVED** [review]

**Review by sshropshire — APPROVED** [review]

**Pull request merged**